packages/core/src/generation.ts

```diff
@@ -479,6 +479,29 @@ export async function generateText({
                 break;
             }
 
+            case ModelProviderName.VENICE: {
```

Comment:
You can just add this as a case around line 149, as venice follow the openAI api spec.

Reply:
Yea I tested this as well (definitely works) but I plan to add the venice_parameters that are exclusive to venice as they come out.